src/style.css

```
@@ -0,0 +1,80 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.searchBar {
+  display: block;
+  box-sizing: border-box;
+  margin: 20px 0;
+  padding-left: 62px;
+  height: 70px;
+  width: 100%;
+  background: #fff;
+  background-image: transparent url("images/Search.svg") no-repeat 26px 25px / 19px 19px;
+
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.searchBar--big::placeholder {
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
```

```suggestion
.searchBar::placeholder {
  font-size: 16px;
  line-height: 22px;
  color: #3d4e61;
}
```
3 point in checklist